content/docs/alerting/configuration.md

```diff
@@ -64,10 +64,16 @@ global:
   # The default SMTP From header field.
   [ smtp_from: <tmpl_string> ]
   # The default SMTP smarthost used for sending emails.
+  # Provide the correct port for the smarthost:
```

Comment:
This is takes up a bit much space for reference docs.  I'd mention that the smarthost must include a port, and quickly run down the ports rather than give them a line each.